app/services/meetup_service.rb

```diff
@@ -0,0 +1,19 @@
+class MeetupService
+  class << self
+    GROUP_NAME = "bostonrb"
+    
+    def fetch_upcoming_event
+      url = "#{base_url}/events"
+      response = HTTParty.get(url, page: 1)
+      MeetupEvent.new(JSON.parse(response.body).first)
+    rescue
```

Comment:
I should probably handle this in the controller itself and not fully swallow the error, but I basically just didn't want the page to blow up if the meetup request failed.